express.js

```diff
@@ -215,11 +215,6 @@ app.get('/account/checkUsername', require('./routes/views/checkUsername'));
 app.get('/password_resetted', require(routes + 'account/get/requestPasswordReset'));
 app.get('/report_submitted', require(routes + 'account/get/report'));
 // Download Client
```

Comment:
This comment can be removed as well correct?

Reply:
done

Reply:
we need to cleanup the express.js so bady, maybe if i got the energy for it.